src/program.js

```diff
@@ -401,7 +401,7 @@ Example: $0 --help run.
                   'run against multiple targets.',
         default: 'firefox-desktop',
         demand: false,
-        type: 'string',
+        type: 'array',
```

Comment:
Can you also update the Flow type:

https://github.com/rpl/web-ext/blob/3848c777b290b4c5ed6ad1af0436afe5f952b005/src/cmd/run.js#L44

It should now just be `target?: Array<string>`.

Reply:
I applied these changes to the flow types in 
025db7c